systemtests/src/test/java/io/enmasse/systemtest/scale/ScaleTest.java

```diff
@@ -257,6 +261,14 @@ void testMessagingPerformance() throws Exception {
             manager.stopMonitoring();
             manager.gatherPerformanceResults();
             saveResultsFile("messaging_performance_results.json", manager.getPerformanceResults());
+            Map<String, Object> data = new HashMap<>();
+            data.put("addresses", addresses.size());
+            data.put("connections", manager.getConnections());
+            data.put("anycastLinksPerConnection", anycastLinksPerConnection);
+            data.put("queueLinksPerConnection", queueLinksPerConnection);
+            data.put("anycastMedianThroughput", manager.getPerformanceResults().getAddresses().get(AddressType.ANYCAST.toString()).getGlobalSenders().getEstimateTotalThroughputMedian());
```

Comment:
here you are only giving results for senders, is important to include receivers

Reply:
ok makes sense